src/ExpressionEvaluator/Package/source.extension.vsixmanifest

```diff
@@ -4,6 +4,7 @@
     <Identity Id="21BAC26D-2935-4D0D-A282-AD647E2592B5" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Expression Evaluators</DisplayName>
     <Description xml:space="preserve">Roslyn Expression Evaluators</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\Expression Evaluators</ShortcutPath>
```

Comment:
@brettfo shoupd confirm but its probably relative to the Common7\IDE\Extensions folder